app/api/v3/token.py

```diff
@@ -0,0 +1,1775 @@
+        transfer_approval_required: Optional[bool] = request_json.get("transfer_approval_required", None)
+        is_redeemed: Optional[bool] = request_json.get("is_redeemed", None)
+
+        sort_item = "created" if request_json["sort_item"] is None else request_json["sort_item"]
+        sort_order = 0 if request_json["sort_order"] is None else request_json["sort_order"]  # default: asc
+        offset = request_json["offset"]
+        limit = request_json["limit"]
+
+        # 取扱トークンリストを取得
+        # 公開属性によるフィルタリングを行うためJOIN
+        query = session.query(IDXBondToken).\
+            join(Listing, Listing.token_address == IDXBondToken.token_address).\
+            filter(Listing.is_public == True)
+        if len(target_address_list) > 0:
+            query = query.filter(IDXBondToken.token_address.in_(target_address_list))
+        total = query.count()
+
+        # Search Filter
+        if owner_address is not None:
+            query = query.filter(IDXBondToken.owner_address == owner_address)
+        if name is not None:
+            query = query.filter(IDXBondToken.name.contains(name))
+        if symbol is not None:
+            query = query.filter(IDXBondToken.symbol.contains(symbol))
+        if company_name is not None:
+            query = query.filter(IDXBondToken.company_name.contains(company_name))
+        if tradable_exchange is not None:
+            query = query.filter(IDXBondToken.tradable_exchange == tradable_exchange)
+        if status is not None:
+            query = query.filter(IDXBondToken.status == status)
+        if personal_info_address is not None:
+            query = query.filter(IDXBondToken.personal_info_address == personal_info_address)
+        if transferable is not None:
+            query = query.filter(IDXBondToken.transferable == transferable)
+        if is_offering is not None:
+            query = query.filter(IDXBondToken.is_offering == is_offering)
+        if transfer_approval_required is not None:
+            query = query.filter(IDXBondToken.transfer_approval_required == transfer_approval_required)
+        if is_redeemed is not None:
+            query = query.filter(IDXBondToken.is_redeemed == is_redeemed)
+        count = query.count()
+
+        sort_attr = getattr(IDXBondToken, sort_item, None)
+
+        if sort_order == 0:  # ASC
+            query = query.order_by(sort_attr)
+        else:  # DESC
+            query = query.order_by(desc(sort_attr))
+        if sort_item != "created":
+            # NOTE: Set secondary sort for consistent results
+            query = query.order_by(IDXBondToken.created)
+
+        # Pagination
+        if limit is not None:
+            query = query.limit(limit)
+        if offset is not None:
+            query = query.offset(offset)
+
+        _token_list: list[IDXBondToken] = query.all()
+        tokens = []
+
+        for _token in _token_list:
+            tokens.append(BondToken.from_model(_token).__dict__)
```

Comment:
Convert model data to existing Token Object so that we could return the same schema as v2/Token use.